File: CCE DataScienceModels/Shared/DSModel/common/log_message_level.py
from enum import Enum

class LogMessageLevel(Enum):
    NONE = 0    # Log no messages
    ERROR = 1   # Error; code is likely to have failed
    WARNING = 2 # Warnings; code will continue
    INFO= 3    # Informative messages; FYI
    DEBUG = 4   # Debug messages; More detail than Info
    TRACE = 5   # Trace messages: Fine-grained trace information
    ALL = 255   # All messages

File: POL_SIL_Furnace/sandbox/Shared/Logger/LogMessageLevel.m
classdef LogMessageLevel < uint8
    enumeration
        None (0)    % Log no messages
        Error (1)   % Error; code is likely to have failed
        Warning (2) % Warnings; code will continue
        Info (3)    % Informative messages; FYI
        Debug (4)   % Debug messages; More detail than Info
        Trace (5)   % Trace messages: Fine-grained trace information
        All (255)   % All messages
    end
end